pkg/domain/resourcegroup/runaway.go

```diff
@@ -127,6 +148,7 @@ func (r *QuarantineRecord) GenInsertionStmt() (string, []any) {
 	params = append(params, r.WatchText)
 	params = append(params, r.Source)
 	params = append(params, r.Action)
+	params = append(params, r.GetSwitchGroupName())
```

Comment:
Does action already contain the switch group name here?

Reply:
No, since `r.Action` of type `rmpb.RunawayAction` only contains the action type information, extending it in `kvproto` could cause backward compatibility issues. Therefore, I chose to add a new column to store the switch group name instead.